src/FluentAssertions.Analyzers.Tests/Tips/CollectionTests.cs

```diff
@@ -414,6 +414,14 @@ public class CollectionTests
         [Implemented]
         public void CollectionShouldNotBeNullOrEmpty_TestCodeFix(string oldAssertion, string newAssertion) => VerifyCSharpFixCodeBlock<CollectionShouldNotBeNullOrEmptyCodeFix, CollectionShouldNotBeNullOrEmptyAnalyzer>(oldAssertion, newAssertion);
 
+        [TestMethod]
+        [Implemented]
+        public void CollectionShouldHaveElementAt_ShouldIgnoreDictionaryTypes()
```

Comment:
I think this should be in the sanity tests and linked to its github issue

Reply:
Not a problem, didn't see that file 🙂